### PR TITLE
Added valentine growl

### PR DESCRIPTION
**Pull request by mzparacha**
## Link to Issue
Closes: https://github.com/hicommonwealth/commonwealth/issues/6636

## Description of Changes
- Added Valentine's growl
image

## "How We Fixed It"

N/A

## Test Plan
- Visit any page
- Verify you see the valentine's growl
- Click on the 'Add to Calender' button, verify it opens OS calender and adds event entry.
- Try to close it from the x button, verify it closes and shows again on page reload
- Click on the "Don't show this again" checkbox, and close it again from the x button, verify it closes and doesn't show again on page reload.

## Deployment Plan
N/A

## Other Considerations
N/A

### CODE REVIEW

File: packages/commonwealth/static/img/valentineGrowlImage.jpeg
Comment (kurtisassad) [inline packages/commonwealth/static/img/valentineGrowlImage.jpeg:1]: Maybe not required to fix now, but we should parking lot in the future about how we should be handling assets. We should not be serving them from our server, but instead pulling them from our s3 through our CDN.

But we don't currently have any pattern code-wise for that, so as I mentioned it probably doesn't need to be fixed in this PR.

**Review by CowMuon — APPROVED** [review]
Really appreciate the detail level in testing instructions, @mzparacha 
This appears to do the thing- can we ship this, with gating growl removal [6517](https://github.com/hicommonwealth/commonwealth/pull/6517) as patch 0.9.4-4 @ilijabojanovic

**Review by kurtisassad — APPROVED** [review]
Looks great!

**Comment by mzparacha** [comment]
@ilijabojanovic can you try on port 8080?

**Comment by kurtisassad** [comment]
For what its worth it works for me, but mine goes through google calendar. Maybe it breaks on the macos calendar app?

**Comment by mzparacha** [comment]
Works for me @ilijabojanovic, I am not sure what the issue on your side is. 

https://github.com/hicommonwealth/commonwealth/assets/51641047/3d439cbd-846d-4e3a-9422-a4ff00faa6de

**Comment by CowMuon** [comment]
Approved to ship, but good catch @ilijabojanovic, this was exactly what we were discussing in parking lot, that it's not uncommon for cal invite handling to have a lacuna between web calendars and locally installed calendar apps. We said on that call we're good with users having to take extra steps. 

I haven't tested on Frick or Frack tho it looks like Ilija's error is local only.

**Comment by mzparacha** [comment]
Pushed to frack for testing

cc: @CowMuon @ilijabojanovic

**Pull request merged**